internal/commands/app/create.go

```diff
@@ -64,20 +64,16 @@ func (cmd *CommandCreate) Handler(profile *user.Profile, ui terminal.UI, clients
 		return err
 	}
 
-	var groupID = cmd.inputs.Project
-	if appRemote.IsZero() {
-		if groupID == "" {
-			id, err := cli.ResolveGroupID(ui, clients.Atlas)
-			if err != nil {
-				return err
-			}
-			groupID = id
+	groupID := cmd.inputs.Project
+	if groupID == "" {
+		id, err := cli.ResolveGroupID(ui, clients.Atlas, appRemote.GroupID)
```

Comment:
instead of adding the `defaultGroupID` as an argument to `ResolveGroupID`, thoughts on using the following approach:

```go
groupID := cmd.inputs.Project
if groupID == "" {
  groupID = appRemote.GroupID
}
if groupID == "" {
  groupID, err = cli.ResolveGroupID(ui, clients.Atlas)
  if err != nil { return err }
}
```

I _think_ that removes the need to modify `ResolveGroupID`, thoughts?

Reply:
The default group was added to account for the auto confirm flag. 
We pass in the group that the remote app belonged to to ResolveGroupID so it could be selected in case the auto confirm flag is set.